src/domain/event/occurrences/Occurrences.tsx

```diff
@@ -231,11 +232,16 @@ const Occurrences: React.FC<Props> = ({
   ];
 
   const renderOccurrenceInfo = (occurrence: OccurrenceFieldsFragment) => {
-    const { placeId, startTime, endTime } = occurrence;
+    const { placeId, startTime, endTime, linkedEvent, pEvent } = occurrence;
     const date = formatDate(new Date(startTime));
     const time =
       startTime &&
       formatTimeRange(new Date(startTime), new Date(endTime), locale);
+    const offer = linkedEvent?.offers?.[0];
+    const price: string = offer?.price?.[locale] || '';
+    const priceDescription: string = offer?.description?.[locale] || '';
+    const isFree: boolean = offer?.isFree || price === '';
+    const paymentInstruction = pEvent?.paymentInstruction || '';
```

Comment:
adding types is adding some unnecessary verbosity, I don't think they are needed 🤔 

if price is string, you could write` !price`  instead of `price === ''`;

also it might be better to use `??` operator instead of `||` even though `||` works fine

Reply:
All suggestions were good and are now changed.

Reply:
not pushed yet?